### ISSUE TITLE
JS Error

### ISSUE THREAD

**Issue by rossdakin** [18F/pif-website#36]
Present on all pages except home page:

<img width="516" alt="screen shot 2015-07-29 at 5 06 43 pm" src="https://cloud.githubusercontent.com/assets/44956/8969816/c477c604-3613-11e5-98bb-913529cd1ae9.png">

Prevents JS things from working, e.g. filters on https://presidentialinnovationfellows.gov/fellows/

Suspect the [video player](https://github.com/18F/pif-website/blob/a6abba005ba27a7fafa558dbecdc06f7ed830c53/js/main.js#L96).

Suggestion (I would open a PR but don't have a dev environment setup yet): maybe just wrap it all in a function and bail near the top if the video element doesn't exist.

E.g. turn this:

```JavaScript
  /* video player */
  var video_player = document.getElementById("video_player");
  var links = video_player.getElementsByTagName('a');
  for (var i=0; i<links.length; i++) {
    links[i].onclick = handler;
  }

  /*** more video stuff ***/
```

Into this:

```JavaScript
  /* video player */
  (function() {
    var video_player = document.getElementById("video_player");

    // bail on non-video pages
    if (!video_player) {
      return;
    }

    var links = video_player.getElementsByTagName('a');
    for (var i=0; i<links.length; i++) {
      links[i].onclick = handler;
    }

    /*** more video stuff ***/
  })();
```

**Comment by stroupaloop** [comment]
hm will dig into this more, but there's another issue #32 around updating the video player to include more functionality that will potentially solve this.

The tutorial provided doesn't fully work, so I have to spend some time digging through the JS to resolve broken things while maintaining current functionality.

**Comment by stroupaloop** [comment]
resolved until upgrade :) 011893d
